docs/release-notes/eclair-vnext.md

```diff
@@ -4,6 +4,31 @@
 
 ## Major changes
 
+### Anchor outputs activated by default
+
+Experimental support for anchor outputs channels was first introduced in [eclair v0.4.2](https://github.com/ACINQ/eclair/releases/tag/v0.4.2).
+Going from an experimental feature to production-ready required a lot more work than we expected!
+What seems to be a simple change in transaction structure had a lot of impact in many unexpected places, and fundamentally changes the mechanisms to ensure funds safety.
+
+When using anchor outputs, you will need to keep utxos available in your `bitcoind` wallet to fee-bump HTLC transactions when channels are force-closed.
+Eclair will warn you via the `notifications.log` file when your `bitcoind` wallet balance is too low to protect your funds against malicious channel peers.
```

Comment:
```suggestion
Eclair will warn you via the `notifications.log` file when your `bitcoind` wallet balance is too low to protect your funds against malicious channel peers. Eclair will wait for you to add funds to your wallet and automatically retry, but you may be at risk in the meantime if you had pending payments at the time of the force-close.
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2134/commits/07da8abdb4d4c124eff134c0347676adc80ff48c